m2cgen/assemblers/tree.py

```diff
@@ -13,7 +16,8 @@ def __init__(self, model):
         super().__init__(model)
         self._tree = model.tree_
         self._is_vector_output = False
-        if isinstance(self.model, tree.DecisionTreeClassifier):
+        if type(self.model).__name__ in {"DecisionTreeClassifier",
+                                         "ExtraTreeClassifier"}:
```

Comment:
Now it's in consistency with a similar check in `boosting` assembler:
https://github.com/BayesWitnesses/m2cgen/blob/4c0f002885c65a2aac7725d64bfe47570b833248/m2cgen/assemblers/boosting.py#L25